README.md

```diff
@@ -186,12 +186,12 @@ Element requires the following URLs not to be cached, when/if you are serving El
 Development
 ===========
 
-Before attempting to develop on Riot you **must** read the [developer guide
+Before attempting to develop on Element you **must** read the [developer guide
 for `matrix-react-sdk`](https://github.com/matrix-org/matrix-react-sdk#developer-guide), which
 also defines the design, architecture and style for Element too.
 
 Before starting work on a feature, it's best to ensure your plan aligns well
-with our vision for Riot. Please chat with the team in
+with our vision for Element. Please chat with the team in
 [#riot-dev:matrix.org](https://matrix.to/#/#riot-dev:matrix.org) before you
```

Comment:
The room alias _has_ changed to `#element-dev`, so likely good to update it.

Reply:
Done!